ci/.travis.yml

```diff
@@ -1,142 +0,0 @@
-#
-# DO NOT EDIT THIS FILE
-#
-# It is automatically copied from https://github.com/pion/.goassets repository.
-# If this repository should have package specific CI config,
-# remove the repository name from .goassets/.github/workflows/assets-sync.yml.
-#
-# If you want to update the shared CI config, send a PR to
-# https://github.com/pion/.goassets instead of this repository.
-#
-
-dist: bionic
-language: go
-
-
-branches:
-  only:
-  - master
-
-env:
-  global:
-    - GO111MODULE=on
-    - GOLANGCI_LINT_VERSION=1.19.1
-
-cache:
-  directories:
-    - ${HOME}/.cache/go-build
-    - ${GOPATH}/pkg/mod
-  npm: true
-  yarn: true
-
-_lint_job: &lint_job
-  env: CACHE_NAME=lint
-  before_install:
-    - if [ -f .github/.ci.conf ]; then . .github/.ci.conf; fi
-  install: skip
-  before_script:
-    - |
-      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
-        | bash -s - -b $GOPATH/bin v${GOLANGCI_LINT_VERSION}
-  script:
-    - bash .github/assert-contributors.sh
-    - bash .github/lint-disallowed-functions-in-library.sh
-    - bash .github/lint-commit-message.sh
-    - bash .github/lint-filename.sh
-    - golangci-lint run ${GOLANGCI_LINT_EXRA_ARGS:-} ./...
-_test_job: &test_job
-  env: CACHE_NAME=test
-  before_install:
-    - if [ -f .github/.ci.conf ]; then . .github/.ci.conf; fi
-    - go mod download
-  install:
-    - go build ./...
-  script:
-    # If you want to specify repository specific test packages rule,
-    # add `TEST_PACKAGES=$(command to list test target packages)` to .github/.ci.conf
-    - testpkgs=${TEST_PACKAGES:-$(go list ./... | grep -v examples)}
-    - coverpkgs=$(echo "${testpkgs}" | paste -s -d ',')
-    - |
-      go test \
-        -coverpkg=${coverpkgs} -coverprofile=cover.out -covermode=atomic \
-        ${TEST_EXTRA_ARGS:-} \
-        -v -race ${testpkgs}
-    - if [ -n "${TEST_HOOK}" ]; then ${TEST_HOOK}; fi
```

Comment:
This also be removed and 
https://github.com/pion/stun/blob/14c00f4f988e234c82b71cde14ba3583037f392b/.github/.ci.conf#L6-L17
is not executed at now.